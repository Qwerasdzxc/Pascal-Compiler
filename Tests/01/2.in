698.072 909.171

