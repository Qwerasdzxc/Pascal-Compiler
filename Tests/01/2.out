1607.24 -211.10 0.77